app/src/main/java/org/p2p/wallet/sell/ui/lock/SellLockedFragment.kt

```diff
@@ -128,4 +120,23 @@ class SellLockedFragment :
             showUiKitSnackBar(messageResId = R.string.common_copied)
         }
     }
+
+    override fun navigateBack() {
+        popBackStackTo(SellPayloadFragment::class)
+    }
+
+    override fun navigateBackToMain() {
+        popBackStackTo(MainFragment::class)
+    }
+
+    private fun showWarningDialog() {
+        MaterialAlertDialogBuilder(requireContext())
```

Comment:
material dialog and Alert dialog has different styles, did you check that it is what we want ?

Reply:
i wrote styles special for MaterialAlertDialog, other dialogs are not modified

Reply:
thanks for that sir )